CondFormats/HGCalObjects/interface/HGCalMappingParameterSoA.h

```diff
@@ -35,6 +35,7 @@ namespace hgcal {
                       SOA_COLUMN(bool, valid),
                       SOA_COLUMN(bool, isHD),
                       SOA_COLUMN(bool, iscalib),
+                      SOA_COLUMN(int, offset),
```

Comment:
I would suggest using a more descriptive name such that it's clear the offset is for the calibration channel, something like `caliboffset`?